src/ui/public/chrome/directives/app_switcher/app_switcher.js

```diff
@@ -1,10 +1,18 @@
 import DomLocationProvider from 'ui/dom_location';
 import { parse } from 'url';
-import { bindKey } from 'lodash';
+import { bindKey, filter } from 'lodash';
 import '../app_switcher/app_switcher.less';
 import uiModules from 'ui/modules';
 import appSwitcherTemplate from './app_switcher.html';
 
+function isNavLinkEnabled(link) {
+  return link.enable === undefined || link.enable;
+}
+
+function isNavLinkShown(link) {
+  return link.show === undefined || link.show;
```

Comment:
@cjcenizal @epixa Another option might be to continue using the positive `show` (or `shown`) and `enable` (or `enabled`) properties but define them explicitly in the `UiNavLink` class, defaulting them both to `true`. And I'd add tests for ensuring these properties are being set correctly and returned in `toJSON()`.

That would let us simplify or even get rid of the `isNavLinkEnabled` and `isNavLinkShown` functions in `app_switcher.js`.

Thoughts?